nominal/core/_clientsbunch.py

```diff
@@ -29,6 +30,21 @@
 )
 from typing_extensions import Self
 
+from nominal.ts import IntegralNanosecondsUTC
+
+
+@dataclass(frozen=True)
+class RequestMetrics:
+    oldest_timestamp_diff_before_request: (
+        float  # Time difference between current time and oldest timestamp before request (seconds)
+    )
```

Comment:
you can use docstrings on attributes too instead of inlining the comment, e.g.

```suggestion
    oldest_timestamp_diff_before_request: float
    """Time difference between current time and oldest timestamp before request (seconds)"""
```